pkg/mcs/resourcemanager/server/manager.go

```diff
@@ -342,7 +345,7 @@ func (m *Manager) backgroundMetricsFlush(ctx context.Context) {
 
 			m.consumptionRecord[name] = time.Now()
 
-		case <-ticker.C:
+		case <-cleanUpTicker.C:
```

Comment:
Do we need to cleanup not available resource group?

Reply:
What does unavailable resource group mean?

Reply:
When a group is recorded and then the user deletes it, but it is still in availableRUCounter.

Reply:
I don't think we should delete it? We can avoid unnecessary computing and gRPC overhead by removing resource groups from the client side timely. Regular deletion seems to be ok here.

Reply:
can delete with cleanUpTicker Regularly